src/auctionManager.js

```diff
@@ -91,6 +91,9 @@ export function newAuctionManager() {
   auctionManager.setStatusForBids = function(adId, status) {
     let bid = auctionManager.findBidByAdId(adId);
     if (bid) bid.status = status;
+
+    const auction = find(_auctions, auction => auction.getAuctionId() === bid.auctionId);
```

Comment:
This setStatusForBids function could get called for other statuses someday. Calling setBidTargeting needs to be done only when `status` is 'BID_TARGETING_SET'

Reply:
Done